src/reference/config/zilla-cli.md

```diff
@@ -24,6 +25,28 @@ The Zilla Runtime command line interface uses the [Zilla Runtime Configuration](
 
 ## Commands
 
+### zilla generate
+
+The `zilla generate` command generates a zilla configuration file from an OpenAPI or AsyncAPI service definition.
+
+The command currently has templates for the following common scenarios:
+- `openapi.http.proxy` - Create a http proxy config based on an OpenAPI service definition.
+- `asyncapi.http.proxy` - Create a http proxy config based on an AsyncAPI service definition.
+- `asyncapi.mqtt.proxy` - Create a mqtt proxy config based on an AsyncAPI service definition.
+
+You have to specify which template to use, the OpenAPI/AsyncAPI service definition as the input file and the output
+file name you want the generated zilla config to be saved.
+
+```bash:no-line-numbers
+zilla generate -t template -i input-file -o output-file
```

Comment:
```suggestion
zilla generate -t <template> -i <input-file> -o <output-file>
```
If these are all required field the convention is to document required options with `<required>` and optional with `[optional]`

Reply:
fixed